docs/source/guide/tutorials/base/how_to_train/instance_segmentation.rst

```diff
@@ -1,2 +1,327 @@
+  +-----------------------+----------------------------------------------------------------+--------------------------+-----------------------------------------------------------------------------------------------+
+  | INSTANCE_SEGMENTATION |    Custom_Counting_Instance_Segmentation_MaskRCNN_ResNet50     |    MaskRCNN-ResNet50     |     otx/algorithms/detection/configs/instance_segmentation/resnet50_maskrcnn/template.yaml    |
+  | INSTANCE_SEGMENTATION | Custom_Counting_Instance_Segmentation_MaskRCNN_EfficientNetB2B | MaskRCNN-EfficientNetB2B | otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/template.yaml |
+  +-----------------------+----------------------------------------------------------------+--------------------------+-----------------------------------------------------------------------------------------------+
+
+2. We need to create 
+an OTX-workspace first.
+
+Let's prepare an OTX instance segmentation workspase running the following command:
+
+.. code-block::
+
+  (otx) ...$ otx build --task instance_segmentation --model MaskRCNN-ResNet50
+
+  [*] Workspace Path: otx-workspace-INSTANCE_SEGMENTATION
+  [*] Load Model Template ID: Custom_Counting_Instance_Segmentation_MaskRCNN_ResNet50
+  [*] Load Model Name: MaskRCNN-ResNet50
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/model.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/data_pipeline.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/tile_pipeline.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/deployment.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/hpo_config.yaml
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/compression_config.json
+  [*] Update data configuration file to: otx-workspace-INSTANCE_SEGMENTATION/data.yaml
+
+  (otx) ...$ cd ./otx-workspace-INSTANCE_SEGMENTATION
+
+It will create **otx-workspace-INSTANCE_SEGMENTATION** with all necessery configs for MobileNet-V3-large-1x, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset.
+
+.. note::
+  Using ``otx train`` with TEMPLATE allows you to run the training directly without ``otx build``.
+
+  However, this requires ``--train-data-roots`` and ``--val-data-roots`` in the command.
+
+  .. code-block::
+
+    (otx) ...$ otx train MaskRCNN-ResNet50 \
+                      --train-data-roots  /home/<username>/training_extensions/tests/assets/car_tree_bug \
+                      --val-data-roots /home/<username>/training_extensions/tests/assets/car_tree_bug \
+                      params --learning_parameters.num_iters 8
+
+  The above command also creates an ``otx-workspace-INSTANCE_SEGMENTATION``, just like running build. This also updates ``data.yaml`` with data-specific commands.
+
+  For more information, see :doc:`quick start guide <../../../get_started/quick_start_guide/cli_commands>`
+
+3. Next, we need to create 
```

Comment:
We already created it with ``otx build``

Reply:
Fixed a misleading sentence.